collector/analyzer/tcpmetricanalyzer/tcp_analyzer.go

```diff
@@ -49,6 +49,14 @@ func (a *TcpMetricAnalyzer) Start() error {
 	return nil
 }
 
+func (a *TcpMetricAnalyzer) ConsumableEvents() []string {
+	ret := make([]string, len(consumableEvents))
+	for key := range consumableEvents {
+		ret = append(ret, key)
+	}
```

Comment:
Why not change consumableEvents into a string list and just remove the double-check in the `ConsumeEvent`?

Reply:
Good idea. Done.